packages/core/types/src/http/order/admin/queries.ts

```diff
@@ -14,6 +14,13 @@ export interface AdminOrderFilters extends FindParams, BaseOrderFilters {
   updated_at?: OperatorMap<string>
 }
 
+export interface AdminOrderItemsFilters extends FindParams {
```

Comment:
nit: using `*Filters` is inconsistent with how other types in the HTTP layer is named. I can see we have a bit of a mix of *Params and *Filters, so not sure what the right one is here. Maybe we can just leave this as is, but something that we need to clean up and align on.

Reply:
yup agree, but since there is more *Filter types in the order module I would prefer to refactor this in a separate PR